### PR TITLE
Firebase Chat Development

### PR DESCRIPTION
**Pull request by CarolineHana**
image

Currently the anonymous login is able to produce a random user id on firebase with hard coded data for the user profile picture and display name, eventually this will be customizable for the user to input a name and select an image

### CODE REVIEW

File: src/main/webapp/firebase-chat.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
+/* eslint-disable */
+const firebaseConfig = {
```
Comment (ciddu) [inline src/main/webapp/firebase-chat.js:2]: Confirming here. Will some one with these details be able to write to the DB? Or are there any ACLs that restrict it?

Comment (ciddu) [inline src/main/webapp/firebase-chat.js:2]: Can you moving this config to firebase_config.js? 

These details can be used for other usecases too right?

File: src/main/webapp/firebase-chat.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  // apiKey: removed ,
+  authDomain: 'lounge-95f01.firebaseapp.com',
+  databaseURL: 'https://lounge-95f01.firebaseio.com',
+  projectId: 'youtube-lounge',
+  storageBucket: 'youtube-lounge.appspot.com',
+  messagingSenderId: '681171972170',
+  appId: '1:681171972170:web:4c6526b8eb788af9d876b3',
+  measurementId: 'G-JSDHBSMHS3',
+};
+firebase.initializeApp(firebaseConfig); // eslint-disable-line no-undef
```
Comment (ciddu) [inline src/main/webapp/firebase-chat.js:12]: Looks like this works without the import lint errors you were seeing previously.

File: src/main/webapp/firebase-chat.js (hunk 1 above)
Comment (CarolineHana) [reply src/main/webapp/firebase-chat.js:2]: firestore has security rules we need to implement to restrict the kinds of requests made to our database documents, we will have to write that logic ourselves eventually

**Pull request merged**